media/transcoder/api/Transcoder.Samples/CreateJobWithConcatenatedInputs.cs

```diff
@@ -0,0 +1,148 @@
+
+using Google.Api.Gax.ResourceNames;
+using Google.Cloud.Video.Transcoder.V1;
+
+public class CreateJobWithConcatenatedInputsSample
+{
+    public Job CreateJobWithConcatenatedInputs(
+        string projectId, string location, string inputUri1, decimal startTimeOffset1, decimal endTimeOffset1, string inputUri2, decimal startTimeOffset2, decimal endTimeOffset2, string outputUri)
+    {
+
+        decimal startTimeOffset1NanoSec = 1000000000 * decimal.Subtract(startTimeOffset1, decimal.Truncate(startTimeOffset1));
+        decimal endTimeOffset1NanoSec = 1000000000 * decimal.Subtract(endTimeOffset1, decimal.Truncate(endTimeOffset1));
+        decimal startTimeOffset2NanoSec = 1000000000 * decimal.Subtract(startTimeOffset2, decimal.Truncate(startTimeOffset2));
+        decimal endTimeOffset2NanoSec = 1000000000 * decimal.Subtract(endTimeOffset2, decimal.Truncate(endTimeOffset2));
+
+        // Create the client.
+        TranscoderServiceClient client = TranscoderServiceClient.Create();
+
+        // Build the parent location name.
+        LocationName parent = new LocationName(projectId, location);
+
+        // Build the job config.
+        VideoStream videoStream0 = new VideoStream
+        {
+            H264 = new VideoStream.Types.H264CodecSettings
+            {
+                BitrateBps = 550000,
+                FrameRate = 60,
+                HeightPixels = 360,
+                WidthPixels = 640
+            }
+        };
+
+        AudioStream audioStream0 = new AudioStream
+        {
+            Codec = "aac",
+            BitrateBps = 64000
+        };
+
+        ElementaryStream elementaryStream0 = new ElementaryStream
+        {
+            Key = "video_stream0",
+            VideoStream = videoStream0
+        };
+
+        ElementaryStream elementaryStream1 = new ElementaryStream
+        {
+            Key = "audio_stream0",
+            AudioStream = audioStream0
+        };
+
+        MuxStream muxStream0 = new MuxStream
+        {
+            Key = "sd",
+            Container = "mp4",
+            ElementaryStreams = { "video_stream0", "audio_stream0" }
+        };
+
+        Input input1 = new Input
+        {
+            Key = "input1",
+            Uri = inputUri1
+        };
+
+        Input input2 = new Input
+        {
+            Key = "input2",
+            Uri = inputUri2
+        };
+
+        EditAtom atom1 = new EditAtom
+        {
+            Key = "atom1",
+            StartTimeOffset = new Google.Protobuf.WellKnownTypes.Duration
+            {
+                Seconds = decimal.ToInt32(decimal.Truncate(startTimeOffset1)),
+                Nanos = decimal.ToInt32(startTimeOffset1NanoSec)
+            },
+            EndTimeOffset = new Google.Protobuf.WellKnownTypes.Duration
+            {
+                Seconds = decimal.ToInt32(decimal.Truncate(endTimeOffset1)),
+                Nanos = decimal.ToInt32(endTimeOffset1NanoSec)
+            }
+        };
+        atom1.Inputs.Add("input1");
```

Comment:
```suggestion
        EditAtom atom1 = new EditAtom
        {
            Key = "atom1",
            StartTimeOffset = new Google.Protobuf.WellKnownTypes.Duration
            {
                Seconds = decimal.ToInt32(decimal.Truncate(startTimeOffset1)),
                Nanos = decimal.ToInt32(startTimeOffset1NanoSec)
            },
            EndTimeOffset = new Google.Protobuf.WellKnownTypes.Duration
            {
                Seconds = decimal.ToInt32(decimal.Truncate(endTimeOffset1)),
                Nanos = decimal.ToInt32(endTimeOffset1NanoSec)
            },
            Inputs = { input1.Key }
        };
```

(Use a collection initializer inside the object initializer, and avoid the repeated string literal by using the key from the existing object.)

Reply:
Done